File: Linguagem Dart/1005.dart
import 'dart:io';

void main()
{
    double A = double.parse(stdin.readLineSync());
    double B = double.parse(stdin.readLineSync());
    
    double MEDIA = ((A * 3.5) + (B * 7.5)) / 11;
    
    print("MEDIA = " + MEDIA.toStringAsFixed(5));
}

File: Liguagem C#/1005.cs
using System; 

class URI {

    static void Main(string[] args) { 
    
    double A = double.Parse(Console.ReadLine());
    double B = double.Parse(Console.ReadLine());
    double M = ((A*3.5)+(B*7.5))/11;
    Console.Write("MEDIA = {0:F5}\n",M);
    
    }

}
